### ISSUE TITLE
Installation error. I utilize Windows 10 and Remote Desktop Protocol (RDP) to log into a headless CentOS 7 server to run minerl.

### ISSUE THREAD

**Issue by zhufq00** [minerllabs/minerl#728]
The error log is
[error.log](https://github.com/minerllabs/minerl/files/12171303/error.log)

the demo.py is
[demo.txt](https://github.com/minerllabs/minerl/files/12171309/demo.txt)

while execute obs = env.reset() the program fails.

<img width="952" alt="1690367905358" src="https://github.com/minerllabs/minerl/assets/37504718/eea19b77-2253-4f87-84c8-f1476880e52f">

I have tried to use docker but also fails as it is no longer maintained.

Any ideas? Thanks

**Comment by zhufq00** [comment]
while I use xvfb-run -a python demo.py to run, the error log is
[error-xvfb.log](https://github.com/minerllabs/minerl/files/12171416/error-xvfb.log)

**Comment by Miffyli** [comment]
Looking at the original error file it says something about access denied to files. Could it be that it is trying to store game world states (or MInecraft stuff) to some directory in the system it does not have access to? Minecraft does create some files, but generally it works without a hitch. xvfb-run does not seem to be issue. 

I do not have good suggestions for other than trying ways of running the code with more priviledges / making sure the code has access to some basic places on the system for storing Minecraft data.

**Comment by zhufq00** [comment]
I finally successfully installed it, and I am happy to share it with you. After I modified the permissions, I ran python demo.py, and the program showed
[error.log](https://github.com/minerllabs/minerl/files/12180042/error.log)
GLFW error 65542: GLX: GLX extension not found.
The reason for the error is that the CUDA version is too low, and it worked successfully after I upgraded my CUDA
Thank you for your help.
